web-content/signup.html

```diff
@@ -149,7 +148,7 @@
             </div>
           </div>
           <div class="control-group">
-            <label class="control-label" for="email">Signup code:</label>
+            <label class="control-label" for="signup_code">Signup code:</label>
             <div class="controls">
               <input type="text" class="input-xlarge" name="signup_code">
               <div class="help-block">
```

Comment:
Change the `for` in the `checkbox` label to `expires-field` rather than `expires` so clicking on it works.